numba/np/arraymath.py

```diff
@@ -918,29 +918,57 @@ def nanmean_impl(a):
     return nanmean_impl
 
 
+@register_jitable
+def compute_sum_of_square_diffs(a,isnan):
+    m = np.nanmean(a)
+    ssd = 0.0
+    count = 0
+    for view in np.nditer(a):
+        v = view.item()
+        if not isnan(v):
+            val = (v.item() - m)
+            ssd += np.real(val * np.conj(val))
+            count += 1
+    return ssd, count
+
+
 @overload(np.nanvar)
-def np_nanvar(a):
+def np_nanvar(a, ddof):
```

Comment:
`ddof` is a kwarg with default value `0`. https://numpy.org/doc/stable/reference/generated/numpy.nanvar.html#numpy.nanvar. The Numba documentation "A guide to using @overload" is probably worth a read as it covers the sort of things that occur in this PR https://numba.readthedocs.io/en/stable/extending/overloading-guide.html